packages/server/src/server.ts

```diff
@@ -136,7 +135,8 @@ app.getKoaApp().use(async (ctx, next) => {
 const logAndHandleResponse = async (methodName: any, methodParams: any, methodFunction: any) => {
   const start = Date.now();
   let ms;
-  const requestId = generateRequestId();
+
+  const requestId = app.getRequestId();
   const requestIdPrefix = requestId ? `[${REQUEST_ID_STRING}${requestId}]` : '';
```

Comment:
Over in server/tests/helpers.utils.ts someone made a nice tiny method `formatRequestIdMessage`. I don't think it's a bad idea to encapsulate this in a method, even of one line.

Reply:
Agree, let's use this method for consistency.

Reply:
Added a formatters.ts to the server package that follows the pattern for use of the formatters in the relay package.  Updated the existing logging in the servers and rateLimit to use it.